packages/suricata.vm/suricata.vm.nuspec

```
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>suricata.vm</id>
+    <version>7.0.10</version>
+    <authors>Open Information Security Foundation</authors>
+    <description>Suricata is a network IDS, IPS and NSM engine developed by the OISF and the Suricata community</description>
```

New description convention as documented in the [wiki](https://github.com/mandiant/VM-Packages/wiki/Chocolatey-packages#description):
> The description should start with the tool name and end with a `.`.

We are improving the description as part of https://github.com/mandiant/VM-Packages/issues/1231, also motivated by the [FLARE-VM installer enhancements (the FLARE-VM installer now show the descriptions)](https://github.com/mandiant/flare-vm/pull/672).
```suggestion
    <description>Suricata is a network IDS, IPS and NSM engine developed by the OISF and the Suricata community.</description>
```